tribe-ext-community-google-maps-options/index.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Plugin Name:     Community Events: Add Google Map Display and Link Options
+ * Description:        Adds the Google Map Display and Google Map Link options in the venue section of Community Event's frontend editor.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Show_Google_Maps
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
```

Comment:
Removed, old habit to keep it, but with class checks and autoloaders it is not needed.